.env-cmdrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Seems like some of your env config work snuck in to this MR. Is there a way to separate these two functionalities?